bootstrap/004_labels.sql

```diff
@@ -203,3 +299,35 @@ EXCEPTION
 END;
 
 CREATE VIEW CATALOG.LABELS IF NOT EXISTS AS SELECT * FROM INTERNAL.LABELS;
+
+CREATE OR REPLACE PROCEDURE INTERNAL.POPULATE_PREDEFINED_LABELS()
+    RETURNS text
+    LANGUAGE SQL
+    EXECUTE AS OWNER
+AS
+BEGIN
+    let outcome string := '';
+
+    BEGIN TRANSACTION;
```

Comment:
I wonder if the combo of a merge into and a cte of all hte labels will be more readable and wouldn't require the explicit transaction. Something like:

``` sql
with foo as (
select * from (values ('Large Results, 'rows_produced > 5000000000'), 'writes', 'query_type in (\'CREATE_TABLE_AS_SELECT\', \'INSERT\')'))
) MERGE INTO internal.predefined_labels t using foo ...
```

Reply:
Good suggestion. I used a MERGE statement with a SELECT ... values as the source.

```
    MERGE INTO internal.predefined_labels t
    USING (
        SELECT *
        from (values
                ('Large Results', null, null, 'rows_produced > 50000000'),
                ('Writes', null, null, 'query_type in (\'CREATE_TABLE_AS_SELECT\', \'INSERT\')')
             )) s (name, grp, rank, condition)

```